web/src/timerender.ts

```diff
@@ -17,7 +17,9 @@ import {user_settings} from "./user_settings";
 
 let next_timerender_id = 0;
 
-export let display_time_zone = new Intl.DateTimeFormat().resolvedOptions().timeZone;
+export let display_time_zone = new Intl.DateTimeFormat()
+    .resolvedOptions()
+    .timeZone.replace("Calcutta", "Kolkata");
```

Comment:
How did you find this one? Do we have any reason to expect this is the only mismatch of this form?

Reply:
I live in this timezone, therefore noticed the bug here. I looked it up and it seems like this is a known issue with this renaming (and couldn't find others).
Our backend correctly expects the new name `Kolkata`, but Chrome stuck to the old British-era `Calcutta`.

It would make more sense to use the hour offset instead of the name, which we currently don't do in the project.

Reply:
I think this warrants a CZO discussion regarding the implementation.

Reply:
Update: Comparing offsets is a perfectly fine way, and there are no issues with DST.